Updated name

MKonopko opened the pull request:
Is this the name we're going with?

JoaoAndreSa commented:
Was this decided during the steering meeting? The name does make more sense, but I am not sure if changing it goes against what we submitted. We would also have to update the GA4GH website accordingly.

MKonopko commented:
@JoaoAndreSa the SC commented that it was hard to tell who the audience was.  This name should clarify it.  JP and I started discussing a name change but never settled on anything.

JoaoAndreSa commented:
I will ask JP first thing on Monday, but for me it looks good. Do you when do we have to 'release' the final version (with the addressed comments of the SC)?

MKonopko commented:
the 7th of October!

On Fri, Sep 13, 2019 at 9:34 AM Joao Andre Gomes de Sa Sousa <
notifications@github.com> wrote:

> I will ask JP first thing on Monday, but for me it looks good. Do you when
> do we have to 'release' the final version (with the addressed comments of
> the SC)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ga4gh/data-security/pull/17?email_source=notifications&email_token=AHERPM74TJTHBW6EUVMP6M3QJNGCNA5CNFSM4IWNEMX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6ULRXA#issuecomment-531151068>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHERPMYACXH7JM5SE2CVMODQJNGCNANCNFSM4IWNEMXQ>
> .
>


-- 
*Melissa Konopko, MBA, PhD*
Technical Program Manager

melissa.konopko@ga4gh.org
T: +44 01223 49 6973
or +1 508 873 0549
www.ga4gh.org
